maps/include/maps/maputils.h

```diff
@@ -8,7 +8,9 @@
 #include <maps/G3SkyMap.h>
 #include <maps/FlatSkyMap.h>
 
-G3SkyMapPtr GetMaskMap(G3SkyMapConstPtr m);
+G3SkyMapPtr GetMaskMap(G3SkyMapConstPtr m, bool zero_nans=false, bool zero_infs=false);
+
+G3SkyMapPtr ApplyMask(G3SkyMapPtr m, G3SkyMapConstPtr mask, bool inverse=false);
```

Comment:
If these are a public API, the header needs comments on what the functions do and what the arguments mean. Even though the first is an existing function, so you aren't entirely on the hook, the new arguments still require explanation.